pkg/cosign/verify_test.go

```diff
@@ -248,9 +250,17 @@ func TestVerifyImageSignatureWithNoChain(t *testing.T) {
 	opts := []static.Option{static.WithCertChain(pemLeaf, []byte{}), static.WithBundle(rekorBundle)}
 	ociSig, _ := static.NewSignature(payload, base64.StdEncoding.EncodeToString(signature), opts...)
 
+	logId, _ := getLogID(sv.Public)
+	ecdsaKey, _ := sv.PublicKey()
+
 	// TODO(asraa): Re-enable passing test when Rekor public keys can be set in CheckOpts,
 	// instead of relying on the singleton TUF instance.
-	verified, err := VerifyImageSignature(context.TODO(), ociSig, v1.Hash{}, &CheckOpts{RootCerts: rootPool})
+	verified, err := VerifyImageSignature(context.TODO(), ociSig, v1.Hash{}, &CheckOpts{
```

Comment:
You have no idea how excited I'm about this :)

Reply:
Hahahah see my TODO comment above too, I'll fix that in a follow-up!

Reply:
Yes, I saw that, and was referring to the TODO :)